packages/web-components/src/components/ic-pagination/readme.md

```diff
@@ -0,0 +1,53 @@
+# ic-pagination
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property              | Attribute                | Description                                                                               | Type                             | Default     |
+| --------------------- | ------------------------ | ----------------------------------------------------------------------------------------- | -------------------------------- | ----------- |
+| `adjacentCount`       | `adjacent-count`         | The number of pages displayed adjacent to the current page.                               | `number`                         | `1`         |
```

Comment:
The AC only list 3 options for this and the boundaryCount prop (0, 1 & 2). If it is set to number, does that mean that other values can be set? Should it either be an enum or just mention 0, 1 & 2 in the description?

Reply:
Counts are now set to a type listed in the prop table as 0 | 1 | 2, haven't changed the description as it should be evident.